docs/guides/how_to_guides/configuring_datasources/how_to_configure_sorting_in_data_connectors.rst

```diff
@@ -0,0 +1,1029 @@
+.. _how_to_guides__how_to_sort_batches:
+
+How to Configure a Data Connector to Sort Batches
+==================================================
+
+.. warning::
+
+  Sorters are configured through a Data Connector, which is a concept that was introduced in the V3 (BatchRequest) API.
+  The following configuration is not supported by the V2 (batch_kwargs) API.
+
+This guide demonstrates how to sort :ref:`batches <reference__core_concepts__glossary__datasources>` of data that Great Expectations can validate from a filesystem using a configured ``DataConnector``.
```

Comment:
Could you explain a bit more about what is being sorted up here at the top? Why would I want to use a sorter? What will happen if I don't configure a sorter?

Reply:
The "real" answer to this is : "Sorting is helpful when running multi-batch expectations and you want more control over the order in which the validations are run" :) but we may have to keep the explanation a bit more generic for now